shared/searchv3/services-filter/index.js

```diff
@@ -0,0 +1,121 @@
+// @flow
+// The filter bar for search. Lets you select a search provider
+import * as Constants from '../../constants/searchv3'
+import React, {Component} from 'react'
+import {Box, Icon, ClickableBox} from '../../common-adapters'
+import {globalColors, globalStyles} from '../../styles'
+import {isMobile} from '../../constants/platform'
+
+import type {IconType} from '../../common-adapters/icon'
+
+type Props = {|
```

Comment:
this is a shorthand for exact object types: https://flow.org/en/docs/types/objects/#toc-exact-object-types